src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -100,13 +112,17 @@ def consume_scenes(scene_list: list) -> list:
 
         scene_data.append(current_scene_data)
 
-    validate_scene_ids(scene_list)
-    validate_start_times(scene_list)
+    errors.extend(validate_scene_ids(scene_list))
+    errors.extend(validate_start_times(scene_list))
 
-    return scene_data
 
+    if errors:
+        errors.insert(0, f'Errors encountered while consuming scene with ID: {scene["scene_id"]}')
 
-def consume_captions(caption_list: list) -> list:
+    return scene_data, errors
+
+
+def consume_captions(caption_list: list) -> tuple:
```

Comment:
maybe if this method takes in a scene id we can link it to all the captions that have errors for that scene

Reply:
I thought about this, if we cant get the spacing to work I'll do this instead

Reply:
Resolving for now